public/stylesheets/results.css

```diff
@@ -1,3 +1,50 @@
 .resultColumn {
   text-align: center;
-}
\ No newline at end of file
+}
+
+.listContainer.col-md-3 {
+  height:100vh;
```

Comment:
it should make the height dynamic so i dont have to set a fixed height, heigh:100% doesnt work http://stackoverflow.com/questions/1575141/make-div-100-height-of-browser-window